### PR TITLE
add support for Alibaba Cloud KMS Secret Manager

### PR DESCRIPTION
**Pull request by yagrxu**
with current version only the followings are supported:
1. 'ACSCurrenit' for version stage is the default value and recommented.
2. AccessKeyId + AccessKeySecret or STS (assume a provided role name using the provided AccessKeyId and AccessKeySecret). ECS_RAM_ROLE and Kube2Ram are not supported yet.
3. secret value as plaintext - parsing secret value as a json object not yet support

### CODE REVIEW

**Comment by yagrxu** [comment]
I have tested locally and in Alibaba Cloud K8s cluster as well. Please review and merge

**Comment by yagrxu** [comment]
@silasbw Could you please kindly review the change.

File: lib/backends/alicloud-secrets-manager-backend.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    const getSecretValueRequest = new GetSecretValueRequest({
+      secretName: key,
+      versionStage: versionStage
+    })
+
+    const client = this._getClient({ specOptions: { roleArn } })
+    const value = await client.getSecretValue(getSecretValueRequest)
+
+    const secret = { value: value.secretData.toString('utf-8') }
+    return JSON.stringify(secret)
```
Comment (Flydiverny) [inline lib/backends/alicloud-secrets-manager-backend.js:48]: This should probably be something like:
```suggestion
    return value.secretData.toString('utf-8')
```

Comment (yagrxu) [reply lib/backends/alicloud-secrets-manager-backend.js]: thank you for the feedback, I accepted and made the update :)

**Comment by yagrxu** [comment]
ready for review @Flydiverny

**Pull request merged**